README.md

```diff
@@ -1,133 +1,133 @@
-
-> You can also refresh the git repositories with the latest version by calling 'make setup_dev_githttp' or 'make setup_dev_gitssh'
-
-## Troubleshooting
-### Error codes
-Error code returned by the bash (echo $?)
-
-| Code | Description | 
-|--- |--- |
-| 0 | Operation completed successfully 
-|2 | Not connected to Azure subscription. You need to logout / login and set the default subscription 
-|10 | Launchpad is installed but no landingzone and action arguments have been set 
-|11 | Landingzone argument set without an action 
-|12 | Landingzone folder does not exist in the rover 
-
-### Purging Docker cache
-You can purge Docker cache running the following command:
-```bash
-docker system prune -a
-```
-
-### Limitations
-
-* You cannot run the rover from the Azure Cloud Shell (including the Windows Terminal Azure Cloud Shell) at the moment.
-* You cannot run the rover from Powershell on Windows
+## Introduction
+The rover is a Docker container in charge of the deployment of the landing zones in your Azure environment.
+
+You have to load the rover with some landing zones to be able to deploy them.
+
+The rover load the landing zones from Github or directly from your local folder if you are building/modifying a new one on your local machine. Start loading the rover with the Github landing zones as it will be the quickest way to start.
+
+When fully loaded the rover will deploy the landing zones from the launchpad you have installed. 
+
+## Pre-requisites
+* To use the rover you need to have Docker engine running on your local machine
+* For Windows users make sure Docker is running in Linux mode.
+* Visual Studio Code
+* Azure CLI must be installed on your local machine and connected to the default subscription you want to deploy the landingzones
+* jq must be installed if you plan to clone a local copy of the landingzone factory
+* git must be installed if you plan to clone local copy of the landing zones
+
+> For a better experience on Windows 10 we recommended using:
+>- Windows Subsystem for Linux v2 (https://docs.microsoft.com/en-us/windows/wsl/wsl2-install)
+>- Chose Ubuntu 18.04 LTS 
+>- Docker Technical Preview for WSL2 (https://docs.docker.com/docker-for-windows/wsl-tech-preview/)
+>- (Optional) Visual Studio Code insider build 
+
+## Install the Azure CAF Rover on your local machine
+
+To install the rover you must have git already installed
+
+```bash
+wget -O - --no-cache https://raw.githubusercontent.com/aztfmod/rover/master/install.sh | bash
+```
+
+
+## Load the rover with landing zones
+
+### Load the rover with the latest public landing zones
+
+Execute the following command from the local shell. Make sure docker engine is running.
+
+For Linux, MacOS or Windows WSL / WSL2
+```bash
+make
+```
+Go to the next section to install the launchpad
+
+
+## Install the launchpad
+A launchpad is required by the rover to coordinate the initial and sub-sequent deployments of the landing  zones.
+
```

Comment:
Should we add a link to Level 0?

Reply:
I propose to wait until we do more work on the level0 in Jan/Feb